web/packages/extension/src/content.ts

```diff
@@ -174,6 +174,7 @@ function isXMLDocument(): boolean {
             autoplay: options.autostart ? "on" : "auto",
             unmuteOverlay: options.autostart ? "hidden" : "visible",
             splashScreen: !options.autostart,
+            preferredRenderer: options.preferredRenderer,
```

Comment:
This line is no longer necessary due to `...options` above.

Reply:
Oh oops, I thought I removed that, one sec